data_cache.py

```
@@ -62,6 +62,39 @@
+                    if time.monotonic() >= deadline:
+                        raise FileLockTimeout(f"Timed out waiting for lock {self._path}")
+                    time.sleep(_MANIFEST_LOCK_POLL_SEC)
+
+        def release(self) -> None:
+            if self._fd is not None:
+                os.close(self._fd)
+                self._fd = None
+            try:
+                self._path.unlink(missing_ok=True)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)